NebulaWorld/SimulatedWorld.cs

```diff
@@ -433,19 +433,24 @@ public static void OnDronesGameTick(long time, float dt)
             using (GetRemotePlayersModels(out var remotePlayersModels))
             {
                 //Update drones positions based on their targets
+                var prebuildPool = GameMain.localPlanet.factory.prebuildPool;
```

Comment:
Hey, this breaks when you leave the planet. Reproduce by just having an empty save game and flying out of the "sphere of influence" of the starter planet.

Reply:
Thanks for catching this, PR to fix it is up.